inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -1215,24 +1208,6 @@ cdef class InferRequest:
     cpdef wait(self, timeout=None):
         cdef int status
         cdef int64_t c_timeout
-        cdef int c_wait_mode
-        if self._py_callback_used:
-            # check request status to avoid blocking for idle requests
```

Comment:
BTW do you have a test where the wait is called on idle request with callback set?

Reply:
Yes https://github.com/openvinotoolkit/openvino/blob/809bb8af2a4347ae22aba0a99bb248aba9c65b0c/inference-engine/ie_bridges/python/tests/test_InferRequest.py#L302-L329